app/src/main/java/at/gammastrahlung/monopoly_app/fragments/AuctionDialogFragment.java

```diff
@@ -0,0 +1,109 @@
+package at.gammastrahlung.monopoly_app.fragments;
+
+import android.app.Dialog;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.appcompat.app.AlertDialog;
+import androidx.fragment.app.DialogFragment;
+
+import at.gammastrahlung.monopoly_app.R;
+import at.gammastrahlung.monopoly_app.activities.BoardActivity;
+import at.gammastrahlung.monopoly_app.game.Bid;
+import at.gammastrahlung.monopoly_app.game.GameData;
+import at.gammastrahlung.monopoly_app.network.MonopolyClient;
+import at.gammastrahlung.monopoly_app.network.WebSocketHandler;
+
+public class AuctionDialogFragment extends DialogFragment implements WebSocketHandler.ResultBidTrigger {
+    private EditText bidInput;
+    private TextView resultTextView;
+    private WebSocketHandler webSocketHandlerforResultBidTrigger;
+    private Button bidButton;
+    private Button noButton;
+
+
+    public static AuctionDialogFragment newInstance() {
+        return new AuctionDialogFragment();
+    }
+
+    @NonNull
+    @Override
+    public Dialog onCreateDialog(Bundle savedInstanceState) {
+        AlertDialog.Builder builder = new AlertDialog.Builder(getActivity());
+        LayoutInflater inflater = getActivity().getLayoutInflater();
+        View view = inflater.inflate(R.layout.fragment_auction, null);
+        webSocketHandlerforResultBidTrigger = new WebSocketHandler();
```

Comment:
WebSocketHandler should not be accessed from from an Activity/Fragment.
It is only used to handle incoming messages.